public/learn/blog/feed.atom

```diff
@@ -0,0 +1,474 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do the RSS and Atom files have to be committed for this to work? It would be nice if they could be generated at build time, since our blog posts right now are made by nontechnical people committing markdown files, and it'd be nice if we could avoid having to run a build + commit these files on those PRs.

Reply:
These files _are_ generated at build time. I committed them because I saw other generated files (like the sitemap) being committed, so I assumed that no build scripts run during deployment. I can look into how we deploy this site, but any hint is appreciated.

Reply:
We can also work by "trial and error": we can remove the generated feeds from the commits, and see if they appear on the site later.

Reply:
Ah sorry, I worded that wrong, I meant deploy time, during which Vercel should run a build... Now that you mention it, our committed version of our sitemap is out of date, but the deployed one is up to date (https://ironfish.network/sitemap-0.xml) So I think the same should hold for RSS + Atom